Week1/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you will work through the following exercise. The expected result is a list of SQL `CREATE TABLE` statements that represent the tables you're being asked to design. We suggest you first think of the design of the tables (names, columns, types) and then proceed to write the SQL statements, that way you are thinking about the problem you're trying to solve (creating a database for a website), rather than the implementation first (creating a set of SQL statements).
+
+### Option 1
+
+Design the tables for a database that represents a website with food recipes. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
+
+- Each recipe should have a name, a category, a list of ingredients, and a list of steps to follow to complete the recipe. Many recipes might share the same ingredients or the same list of steps (For example, cook pasta according to the package instructions).
+- The website will show a general page with all the recipes.
+- The website will also show recipes under a single category (Salads, Mexican, etc).
+
+### Option 2
+
+Design the tables for a database that represents a website for movies. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
```

Comment:
IMHO, they should at least write `CREATE TABLE` queries/statements.

Reply:
Yes, I have already specified that here: https://github.com/HackYourFuture/databases/pull/500/files#diff-f7c6dc5181d1cbad80620cf352d08e5da13af116551af8d4ab4a4c3532c00c11R1